CPAC/anat_preproc/anat_preproc.py

```diff
@@ -1323,7 +1323,29 @@ def freesurfer_fsl_brain_connector(wf, cfg, strat_pool, pipe_num, opt):
 
     wf.connect(combine_mask, "out_file", binarize_combined_mask, "in_file")
 
-    return wf, {"space-T1w_desc-brain_mask": (binarize_combined_mask, "out_file")}
+    # CCS brain mask is in FS space, transfer it back to native T1 space
+    match_fov_ccs_brain_mask = pe.Node(
+        interface=fsl.FLIRT(), name=f"match_fov_CCS_brain_mask_{node_id}"
+    )
+    match_fov_ccs_brain_mask.inputs.apply_xfm = True
+    match_fov_ccs_brain_mask.inputs.uses_qform = True
+    match_fov_ccs_brain_mask.inputs.interp = "nearestneighbour"
+
+    node, out = strat_pool.get_data("pipeline-fs_raw-average")
+    convert_fs_T1_to_nifti = pe.Node(
+        Function(
+            input_names=["in_file"], output_names=["out_file"], function=mri_convert
+        ),
+        name=f"convert_fs_T1_to_nifti_for_ccs_{node_id}",
+    )
+    wf.connect(node, out, convert_fs_T1_to_nifti, "in_file")
+    wf.connect(
+        convert_fs_T1_to_nifti, "out_file", match_fov_ccs_brain_mask, "reference"
+    )
```

Comment:
```suggestion
    match_fov_ccs_brain_mask = pe.Node(
        interface=fsl.FLIRT(
            apply_xfm=True,
            uses_qform=True,
            interp="nearestneighbour",
            output_type="NIFTI_GZ",
        ),
        name=f"match_fov_CCS_brain_mask_{node_id}",
    )
    node, out = strat_pool.get_data("pipeline-fs_raw-average")
    wf.connect(node, out, match_fov_ccs_brain_mask, "reference")
```

We can just set `output_type="NIFTI_GZ"` in the `nipype.interfaces.fsl.FLIRT` interface instead of outputting an MGZ and converting it.

Reply:
The  reference is in MGZ so FLIRT wont work with Input as NIFTI and Ref as MGZ file, we could use `mri_vol2vol` instead but, FLIRT may be better?

Reply:
Where is the reference coming from that it has to be MGZ and can't be NIfTI?

Reply:
They are ingressed from Freesurfer outputs and reoriented with the node I added before. But they are converted into NIFTI in different stages so I left them MGZ during ingress.